jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -492,16 +494,13 @@ case "$ACTION" in
 
         touch "$JETTY_PID"
         chown "$JETTY_USER" "$JETTY_PID"
-        # FIXME: Broken solution: wordsplitting, pathname expansion, arbitrary command execution, etc.
         su - "$JETTY_USER" $SU_SHELL -c "
           cd \"$JETTY_BASE\"
           echo ${RUN_ARGS[*]} start-log-file=\"$JETTY_START_LOG\" | xargs ${JAVA} > /dev/null &
-          disown \$!
-          echo \$! > \"$JETTY_PID\""
+          disown $(pgrep -P $!)"
       else
+        # Startup if not switching users
         echo ${RUN_ARGS[*]} | xargs ${JAVA} > /dev/null &
-        disown $!
```

Comment:
Since `disown` ha been re-introduced in the branch above, should be re-introduced here as well?